docs/asciidoc/faq.adoc

```diff
@@ -123,7 +123,14 @@ If, on the client side, you observe `Connection prematurely closed BEFORE respon
 * Obtain a https://en.wikipedia.org/wiki/Tcpdump[TCP dump] and check which peer sends a FIN/RST signal.
 * Check your network connection.
 * Check your Firewall and VPN.
-* Check for any proxies and load balancers. Do they have some kind of idle timeout configuration (the connection is closed when there is no incoming data for a certain period of time)?
+* Check for any proxies and load balancers.
+** Do they have some kind of idle timeout configuration (the connection is closed when there is no incoming data for a certain period of time)?
+** Do they silently drop the idle connections without sending any signal?
+In order to verify whether this might be the issue, you can enable the TCP keep-alive as described in the section <<Connection Timeout>>.
```

Comment:
as it is, this sentence is put on the same line as the previous bullet point (see rich diff). since it can technically apply to both bullet point, I'd suggest putting it on its own line by adding `**`

Reply:
It doesn't apply for the first bullet for idle timeout configuration. `idle timeout configuration` and `TCP keep-alive` are two different configurations.